Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserModel.h

```diff
@@ -84,10 +85,14 @@ namespace AzToolsFramework
             AZStd::shared_ptr<RootAssetBrowserEntry> GetRootEntry() const;
             void SetRootEntry(AZStd::shared_ptr<RootAssetBrowserEntry> rootEntry);
 
+            AssetBrowserFilterModel* GetFilterModel() const;
```

Comment:
update: This function should not be marked `const` as we're returning a non-const pointer. Either `GetFilterModel` should be non `const` or we should have an overload that is both `const` and returns a `const` pointer